hummingbird/ml/operator_converters/_array_feature_extractor_implementations.py

```diff
@@ -36,3 +36,18 @@ def forward(self, x):
             return x[:, self.min : self.max]
         else:
             return torch.index_select(x, 1, self.column_indices)
+
+
+class VectorAssemblerModel(BaseOperator, torch.nn.Module):
+    """
+    Class implementing ArrayFeatureExtractor in PyTorch
+
+    This is used by Spark-ML VectorAssembler
+    """
+
+    def __init__(self):
+        super(VectorAssemblerModel, self).__init__(transformer=True)
+
+    def forward(self, *x):
```

Comment:
I think that we can point to concat in `pipeline_implementations.py` for this one.